src/panels/KaitoPanel.ts

```diff
@@ -190,6 +174,51 @@ export class KaitoPanelDataProvider implements PanelDataProvider<"kaito"> {
             return this.resourceManagementClient.resources.getById(this.armId, "2023-08-01");
         });
 
+        // Install kaito enablement
+        const kaitoInstallationResult = await longRunning(`Enabling the kaito for this cluster.`, () => {
```

Comment:
💡 idea: shall we add `${clusterName}` here to be more explicit, just some thoughts?